src/build/patches.ts

```diff
@@ -0,0 +1,77 @@
+import { parse } from "kdljs";
+import { Enum } from "./types";
+import { readdir, readFile } from "fs/promises";
+import { merge } from "./helpers.js";
+
+/**
+ * Converts patch files in KDL to match the [types](types.d.ts).
+ */
+export function parseKDL(kdlText: string) {
```

Comment:
```suggestion
function parseKDL(kdlText: string) {
```

(I don't think this function is used anywhere else)